erc/ERCS/erc-7562.md

```diff
@@ -230,63 +329,66 @@ This means that a `Paymaster` and `Factory` contracts cannot practically be an "
+  `opsAllowed = SAME_UNSTAKED_ENTITY_MEMPOOL_COUNT + (inclusionRate * INCLUSION_RATE_FACTOR) + (min(opsIncluded, 10000)`.
+    * This is a default of `SAME_UNSTAKED_ENTITY_MEMPOOL_COUNT` for a new entity
 * **[UREP-030]** If an unstaked entity causes an invalidation of a bundle, its `opsSeen` is set to `1000`, effectively blocking it from inclusion for 24 hours.
 
 ### Alt-mempools rules:
 
-Alternate mempool is an agreed-upon rule that the bundlers may opt in to.
+Alternate mempool is an agreed-upon rule that the bundlers may opt into, in addition to the canonical mempool
+The alt-mempool "topic" is a unique identifier. By convention, this is the IPFS hash of the document describing (in clear test and YAML file) the specifics of this alt mempool
 
 * **[ALT-010]** The bundler listens to the alt-mempool "topic" over the P2P protocol
 * **[ALT-020]** The alt mempool rules MUST be checked only when a canonical rule is violated
     * That is, if validation follows the canonical rules above, it is not considered part of an alt-mempool.
+*  **[ALT-021]** Such a `UserOperation` (that violates the cannonical rules) is checked against all the "alternate mempools", and is considered part of all those alt-mempools 
 * **[ALT-030]** Bundlers SHOULD forward `UserOperations` to other bundlers only once, regardless of how many (shared) alt-mempools they have. \
-The receiving bundler validates the `UserOperations`, and based on the above rules (and subscribed alt-mempools) decides for which alt-mempools to "tag" it.
+  The receiving bundler validates the `UserOperations`, and based on the above rules (and subscribed alt-mempools) decides which alt-mempools to propagate it to.
 
+### Alt-mempool reputation:
 
-## Security Considerations
 
-### Possible Attacks
+* **[AREP-010]** each alt-mempool manages "opsSeen" and "opsIncluded", much like entities. The opsSeen is incremented after `UserOperation` initial validation, where it is considered part of this mempool.
+    The "opsIncluded" is incremented after this UserOperation is included on-chain (either by this bundler, or another)
+* **[AREP-020]** the alt-mempool becomes THROTTLED based on the [Reputation Calculation](#reputation-calculation) 
+* **[AREP-030]** if such a UserOperation that passed initial validation, later causes a bundle to revert, it is immediately marked as BANNED, by setting its opsIncluded to 0 and opsSeen to 10000
 
-Below is a list of possible attacks that were considered and a reference to the above rule that prevents it.
+## Security Considerations
 
-This list does not attempt to be an exhaustive list of attacks.
-These attacks are examples provided to describe and rationalise the reason for the above rules.
+This document describes the security considerations bundlers must take to protect themselves (and the entire mempool network)
+from denial-of-service attacks.
 
 #### Sample Known (expensive) attack:
```

Comment:
@drortirosh I still think this ENTIRE section (from here till line 446) should be removed.  If we don't remove it then we need to rewrite it, address all the other issues I raised in it, and add all the "trivial" attacks we block.